lib/rack/timeout/core.rb

```diff
@@ -74,8 +74,15 @@ def initialize(app, service_timeout:nil, wait_timeout:nil, wait_overtime:nil, se
       @service_past_wait = service_past_wait == "not_specified" ? ENV.fetch("RACK_TIMEOUT_SERVICE_PAST_WAIT", false).to_s != "false" : service_past_wait
 
       Thread.main['RACK_TIMEOUT_COUNT'] ||= 0
-      if @term_on_timeout
-        raise "Current Runtime does not support processes" unless ::Process.respond_to?(:fork)
+      if @term_on_timeout && !::Process.respond_to?(:fork)
+        raise(NotImplementedError, <<-MSG)
+Current Runtime does not support processes. 
+This probably means that the platform is Windows.
```

Comment:
Also JVM doesn’t support fork. I think it’s okay to either emit this line or make it a bit more open ended like:

> The platform running your application does not support forking (I.e. windows, JVM, etc)


If I’m not on windows and get this message I think it would be confusing why I’m getting it.

Reply:
Cool I'll update for that. Should we also (in another PR) add tests to the matrix for JRuby?

Reply:
i don’t think we need to. In my experience as long as we aren’t using any native dependencies (c code) and we are mindful of fork that’s enough. 

I used to add jruby to my Travis suites back in the day and mostly I found myself spending time fixing the jruby test suite on CI even though the gem code wasn’t actually broken.